src/components/Identity/index.js

```diff
@@ -78,6 +78,7 @@ const createIdentity = ({
     awaitIdentityCookie,
     logger
   });
+  const fireReferrerHideableImage = injectFireReferrerHideableImage();
```

Comment:
This will potentially create 2 iframes. One for Audiences, and one for Identity. We could move this into core and then just pass `fireReferrerHideableImage` as a component tool.